src/electron/platform/android/atfa-scan-results-to-unified-results.ts

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { UnifiedResult } from 'common/types/store-data/unified-data-interface';
+import { UUIDGenerator } from 'common/uid-generator';
+import {
+    AndroidScanResults,
+    BoundingRectangle,
+    RuleResultsData,
+} from 'electron/platform/android/android-scan-results';
+import {
+    AccessibilityHierarchyCheckResult,
+    AtfaBoundingRectangle,
+    SpannableString,
+    ViewHierarchyElement,
+} from 'electron/platform/android/atfa-data-types';
+import { RuleInformation } from 'electron/platform/android/rule-information';
+import { RuleInformationProviderType } from 'electron/platform/android/rule-information-provider-type';
+
+const includedResults = ['ERROR', 'WARNING'];
+
+export function convertAtfaScanResultsToUnifiedResults(
+    scanResults: AndroidScanResults,
+    ruleInformationProvider: RuleInformationProviderType,
+    uuidGenerator: UUIDGenerator,
+): UnifiedResult[] {
+    if (!scanResults || !scanResults.ruleResults) {
+        return [];
+    }
+
+    const unifiedResults: UnifiedResult[] = [];
+
+    for (const atfaResult of scanResults.atfaResults) {
+        if (includeBasedOnResultType(atfaResult)) {
+            const viewElement: ViewHierarchyElement =
+                atfaResult['AccessibilityHierarchyCheckResult.element'];
+            if (viewElement) {
+                const ruleInformation: RuleInformation = ruleInformationProvider.getRuleInformation(
+                    atfaResult['AccessibilityHierarchyCheckResult.checkClass'],
+                );
+
+                if (ruleInformation) {
+                    unifiedResults.push(
+                        createUnifiedResult(
+                            ruleInformation,
+                            atfaResult,
+                            viewElement,
+                            uuidGenerator,
+                        ),
+                    );
+                }
+            }
+        }
+    }
+
+    return unifiedResults;
+}
+
+function createUnifiedResult(
+    ruleInformation: RuleInformation,
+    atfaResult: AccessibilityHierarchyCheckResult,
+    viewElement: ViewHierarchyElement,
+    uuidGenerator: UUIDGenerator,
+): UnifiedResult {
+    const ruleResult: RuleResultsData = {
+        axeViewId: `atfa-${viewElement['ViewHierarchyElement.id']}`,
```

Comment:
It feels strange to have "axe" in the property name. Maybe in a future PR, we could look into changing this to just `viewId`, since RuleResultsData can refer to either axe or ATFA rule result.

Reply:
`axeViewId` was inherited from the Axe-Android data structure. I like the idea of renaming this in a future PR